src/AppInstallerCLIPackage/Package.appxmanifest

```diff
@@ -1,44 +1,62 @@
-        </uap5:Extension>
-      </Extensions>
-    </Application>
-  </Applications>
-  <Capabilities>
-    <rescap:Capability Name="runFullTrust" />
-    <rescap:Capability Name="packageManagement" />
-  </Capabilities>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package 
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3" 
+  xmlns:uap5="http://schemas.microsoft.com/appx/manifest/uap/windows10/5"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  xmlns:com2="http://schemas.microsoft.com/appx/manifest/com/windows10/2"
+  IgnorableNamespaces="uap uap3 uap5 rescap">
+  <Identity Name="WinGetDevCLI" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.0.16.0" />
+  <Properties>
+    <DisplayName>WinGet Dev CLI</DisplayName>
+    <PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
+  <Dependencies>
+    <!-- Minimum supported version is 1709 (Fall Creators Update, aka RS3) -->
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.16299.0" MaxVersionTested="10.0.19033.0" />
+    <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.25426.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
+  </Dependencies>
+  <Resources>
+    <Resource Language="x-generate" />
+  </Resources>
+  <Applications>
+    <Application Id="WinGetDev" Executable="AppInstallerCLI\AppInstallerCLI.exe" EntryPoint="Windows.FullTrustApplication">
+      <uap:VisualElements DisplayName="WinGet Dev Client" Square150x150Logo="Images\Square150x150Logo.png" Square44x44Logo="Images\Square44x44Logo.png" Description="The WinGet dev client." BackgroundColor="#0078d7" AppListEntry="none" />
+      <Extensions>
+        <uap3:Extension Category="windows.appExtensionHost">
+          <uap3:AppExtensionHost>
+            <uap3:Name>com.microsoft.winget.source</uap3:Name>
+          </uap3:AppExtensionHost>
+        </uap3:Extension>
+        <uap5:Extension Category="windows.appExecutionAlias">
+          <uap5:AppExecutionAlias>
+            <uap5:ExecutionAlias Alias="WinGetDev.exe" />
+          </uap5:AppExecutionAlias>
+        </uap5:Extension>
+        <com:Extension Category="windows.comServer">
+          <com:ComServer>
+            <com:ExeServer Executable="AppInstallerCLIInterfaceServer\AppInstallerCLIInterfaceServer.exe" DisplayName="AppInstaller Interface Server" LaunchAndActivationPermission="O:SYG:SYD:(A;;11;;;WD)(A;;11;;;RC)(A;;11;;;AC)(A;;11;;;AN)S:P(ML;;NX;;;S-1-16-0)">
+              <com:Class Id ="C53A4F16-787E-42A4-B304-29EFFB4BF597" DisplayName="AppInstaller Server">

```

Comment:
>C53A4F16-787E-42A4-B304-29EFFB4BF597 [](http://example.com/codeflow?start=30&length=36)

This is going to conflict with the real package.  Is there any way to make this a separate interface, like we do with `wingetdev`?  I don't really see how we can acheive that, but throwing it out there in case someone else has ideas.

Reply:
Two possible solutions discussed:
1. `#ifdef` our way to having two sets of GUIDs and a duplicate IDL file.
2. Similar, but simply support both sets of GUIDs all of the time.

Reply:
Option 2 is working in the latest version of this. AppInstaller package would register the other set of guids in the appxmanifest.

Reply:
How do we generate the two different headers then?